src/resources/Incorporations/communityContributionCompany.ts

```diff
@@ -7,10 +7,8 @@ export const CommunityContributionCompanyResource: IncorporationResourceIF = {
   entityType: CorpTypeCd.BC_CCC,
   displayName: GetCorpFullDescription(CorpTypeCd.BC_CCC),
   title: 'Community Contribution Company Statement',
-  description: 'This company is a community contribution company, and, as such, has purposes beneficial to ' +
-    'society. This company is restricted, in accordance with Part 2.2 of the BCA, in its ability to pay ' +
-    'dividends and to distribute its assets on dissolution or otherwise.',
-  statement: null,
+  description: '',
+  statement: '',
```

Comment:
I recommend using `null` to indicate that the fields at empty.

Or could you leave the fields out altogether? You might have to update the interface to denote that they're optional.

Reply:
It's done.